memor/prompt.py

```diff
@@ -82,29 +76,27 @@ def __init__(
                 self.update_template(template)
             self.select_response(index=self._selected_response_index)
 
-    def __eq__(self, other_prompt):
+    def __eq__(self, other_prompt: Any) -> bool:
         """
         Check prompts equality.
 
         :param other_prompt: another prompt
-        :type other_prompt: Prompt
-        :return: result as bool
         """
         if isinstance(other_prompt, Prompt):
             return self._message == other_prompt._message and self._responses == other_prompt._responses and \
                 self._role == other_prompt._role and self._template == other_prompt._template and \
                 self._tokens == other_prompt._tokens
         return False
 
-    def __str__(self):
+    def __str__(self) -> str:
         """Return string representation of Prompt."""
         return self.render(render_format=RenderFormat.STRING)
 
-    def __repr__(self):
+    def __repr__(self) -> str:
         """Return string representation of Prompt."""
         return "Prompt(message={message})".format(message=self._message)
 
-    def __copy__(self):
+    def __copy__(self) -> Any:
```

Comment:
Prompt

Reply:
Since it's in the same class it can not be used.

Reply:
I will use following solution: 
[link-to-stackoverflow](https://stackoverflow.com/questions/41135033/type-hinting-within-a-class)